src/UI/Art/SpriteArtist/SpriteOptions.java

```diff
@@ -24,9 +27,9 @@
   private static Dimension dimensions = new Dimension(WINDOWWIDTH_DEFAULT * drawScale, WINDOWHEIGHT_DEFAULT * drawScale);
 
   // Set up configurable options.
-  private static GameOption drawScaleOption = new GameOption("Draw Scale", 1, 6, 1, DRAWSCALE_DEFAULT);
-  private static GameOption animationsOption = new GameOption("Animations", true);
-  private static GameOption damageSystemOption = new GameOption("Damage System", Weapon.stratDescriptions, 0);
+  private static GameOption<Integer> drawScaleOption = new GameOptionInt("Draw Scale", 1, 6, 1, DRAWSCALE_DEFAULT);
+  private static GameOptionBool animationsOption = new GameOptionBool("Animations", true);
```

Comment:
Mild inconsistency here

Reply:
Yay flexibility!

Reply:
Yay flexibility?

Reply:
`GameOptionInt` is the same as `GameOption<Integer>`; use whichever you fancy.

I guess I can make it uniform if you really want.